evennia/utils/evform.py

```diff
@@ -153,6 +153,31 @@
 _ANSI_ESCAPE = re.compile(r"\|\|")
 
 
+def _to_rect(lines):
+    """
+    Forces all lines to be as long as the longest
+
+    Args:
+        lines (list): list of `ANSIString`s
+
+    Returns:
+        nlines (list): list of `ANSIString`s of
+        same length as the longest input line
+
+    """
+    maxl = 0
+    for line in lines:
+        if isinstance(line, (ANSIString, basestring)):
+            maxl = max(len(line), maxl)
+        else:
+            raise ValueError()
+    nlines = []
+    for line in lines:
+        line += ' ' * (maxl - len(line))
+        nlines.append(line)
+    return nlines
```

Comment:
This is fine and quite readable. Just pointing out that this too could be done slightly more succinctly (and more efficiently) using a list comprehension: 

```python
return [line + ' ' * (maxl - len(line)) for line in lines]
```

Reply:
II was going with 

> Simple is better than complex.
[PEP-0020](https://www.python.org/dev/peps/pep-0020/)

Reply:
The difference is nice though 👍 

```diff
diff --git a/evennia/utils/evform.py b/evennia/utils/evform.py
index 8a9828bc..3742f50d 100644
--- a/evennia/utils/evform.py
+++ b/evennia/utils/evform.py
@@ -161,21 +161,12 @@ def _to_rect(lines):
         lines (list): list of `ANSIString`s
 
     Returns:
-        nlines (list): list of `ANSIString`s of
+        (list): list of `ANSIString`s of
         same length as the longest input line
 
     """
-    maxl = 0
-    for line in lines:
-        if isinstance(line, (ANSIString, basestring)):
-            maxl = max(len(line), maxl)
-        else:
-            raise ValueError()
-    nlines = []
-    for line in lines:
-        line += ' ' * (maxl - len(line))
-        nlines.append(line)
-    return nlines
+    maxl = max(len(line) for line in lines)
+    return [line + ' ' * (maxl - len(line)) for line in lines]

```